content/W2016/W16-event-blogging.md

```diff
@@ -1,5 +1,5 @@
 Title: Big CSters Blogging Workshop
-Date: 2016-02-24 14:00
+Date: 2016-02-29 14:00
```

Comment:
I think there there a "updated" metadatum you can use instead of changing history.